app/feedpeople/urls.py

```diff
@@ -0,0 +1,3 @@
+from django.urls import path
+
+urlpatterns = []
```

Comment:
As noted above, this file can be removed.

Reply:
Ok thanks!